js/block.js

```diff
@@ -1766,66 +1776,67 @@ class Block {
     }
 
     // Utility functions
-    isValueBlock() {
-        return this.protoblock.style === "value";
-    }
 
-    isNoHitBlock() {
-        return NOHIT.indexOf(this.name) !== -1;
-    }
+    //Changed to arrow
```

Comment:
Not that it impacts the code, but changing the order of these functions makes it harder to review.